src/DI/Container.php

```diff
@@ -14,14 +15,14 @@
 /**
  * @implements ArrayAccess<string,mixed>
  */
-class Container implements ArrayAccess
+final class Container implements ArrayAccess
```

Comment:
I've been keeping this none final as Minicli extended it

Reply:
I’ll need to update the pint config because it’s the pint command that’s adding this

Reply:
It should be easy enough to make this abstract as we never directly work with the container 🙂

I "believe" there's an option to skip final on internal classes - but cannot remember how to configure it!